pycbc/population/live_pastro_utils.py

```diff
@@ -71,6 +76,32 @@ def __init__(self, specfile, bank):
             self.spec = _check_spec[self.method](self.spec_json)
             self.bank = _read_bank[self.method](self.spec, bank)
 
+    def apply_significance_limits(self, trigger_data):
+        """
+        If the network SNR and FAR indicate saturation of the FAR estimate,
+        set them to the fixed values given in the specification.
+        """
+        # This only happens for double or triple events
+        if len(trigger_data['triggered']) == 1:
+            return trigger_data
+
+        if len(trigger_data['triggered']) > 1:
+            farlim = self.spec['limit_far']
+            snrlim = self.spec['limit_snr']
+            # Only do anything if FAR and SNR are beyond given limits
+            if trigger_data['far'] > farlim or \
+                    trigger_data['network_snr'] < snrlim:
+                return trigger_data
+
+            logging.debug('Truncating FAR and SNR from %f, %f to %f, %f',
+                          trigger_data['far'], trigger_data['network_snr'],
+                          farlim, snrlim)
+            trigger_data['network_snr'] = snrlim
+            trigger_data['far'] = farlim
+            return trigger_data
+
+        raise RuntimeError('Number of triggered ifos must be >0 !')
```

Comment:
I am pretty sure `len()` will not return a number less than zero, so this should look like this:
```suggestion
        farlim = self.spec['limit_far']
        snrlim = self.spec['limit_snr']
        # Only do anything if FAR and SNR are beyond given limits
        if trigger_data['far'] > farlim or \
                trigger_data['network_snr'] < snrlim:
            return trigger_data

        logging.debug('Truncating FAR and SNR from %f, %f to %f, %f',
                      trigger_data['far'], trigger_data['network_snr'],
                      farlim, snrlim)
        trigger_data['network_snr'] = snrlim
        trigger_data['far'] = farlim
        return trigger_data
```

Reply:
No, the error is in the (unexpected) case where it is an empty list with length 0, which is not negative but also not >0.